docs/tools.rst

```diff
@@ -10,6 +10,17 @@ ______________________________
   Database (IEDB): 2018 update. Nucleic Acids Res. 2018 Oct 24.
   doi: `10.1093/nar/gky1006 <10.1093/nar/gky1006>`_. [Epub ahead
   of print] PubMed PMID: `30357391 <https://www.ncbi.nlm.nih.gov/pubmed/30357391>`_.
+- License: Non-Profit OSL 3.0
+
+  By using the IEDB software, you are consenting to be bound by and become a
+  "Licensee" for the use of IEDB tools and are consenting to the terms and
+  conditions of the Non-Profit Open Software License ("Non-Profit OSL") version 3.0.
+
+  Please read these two license agreements `here <http://tools.iedb.org/mhci/download/>`_
+  before proceeding. If you do not agree to all of the terms of these two agreements,
+  you must not install or use the product. Companies (for-profit entities) interested
+  in downloading the command-line versions of the IEDB tools or running the entire analysis
+  resource locally, should contact us (license@iedb.org) for details on licensing options.
```

Comment:
Is this copy-pasted from IEDB?  Potential users probably should not contact `us`, but `IEDB` instead.

Reply:
Fixed.